03/11/2020 07:48:10 - WARNING - __main__ -   Process rank: -1, device: cpu, n_gpu: 0, distributed training: False, 16-bits training: False
{'cola': <class 'transformers.data.processors.glue.ColaProcessor'>, 'mnli': <class 'transformers.data.processors.glue.MnliProcessor'>, 'mnli-mm': <class 'transformers.data.processors.glue.MnliMismatchedProcessor'>, 'mrpc': <class 'transformers.data.processors.glue.MrpcProcessor'>, 'sst-2': <class 'transformers.data.processors.glue.Sst2Processor'>, 'sts-b': <class 'transformers.data.processors.glue.StsbProcessor'>, 'qqp': <class 'transformers.data.processors.glue.QqpProcessor'>, 'qnli': <class 'transformers.data.processors.glue.QnliProcessor'>, 'rte': <class 'transformers.data.processors.glue.RteProcessor'>, 'wnli': <class 'transformers.data.processors.glue.WnliProcessor'>}
Traceback (most recent call last):
  File "/scratch/fs45/nlu/code/transformers/examples/run_glue.py", line 702, in <module>
    main()
  File "/scratch/fs45/nlu/code/transformers/examples/run_glue.py", line 611, in main
    raise ValueError("Task not found: %s" % (args.task_name))
ValueError: Task not found: boolq
